docs/rules/prefer-separate-static-class.md

```diff
@@ -0,0 +1,43 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/prefer-separate-static-class
+description: require static class names in template to be in a separate `class` attribute
+---
+# vue/prefer-separate-static-class
+
+> require static class names in template to be in a separate `class` attribute
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule reports static class names in dynamic class attributes.
+
+<eslint-code-block fix :rules="{'vue/prefer-separate-static-class': ['error']}">
+
+```vue
+<template>
+  <!-- FAIL -->
```

Comment:
I like to use the same comments as other documents.

```suggestion
  <!-- ✗ BAD -->
```

Reply:
Fixed in [`031ba99`](https://github.com/vuejs/eslint-plugin-vue/pull/1729/commits/031ba99c1fc830383418592d9478fbf743f815cd).